src/sass/edu-benefits.scss

```diff
@@ -7,88 +7,20 @@
 @import "modules/m-schemaform";
 @import "modules/m-modal";
 @import "modules/m-omb-info";
+@import "modules/m-form-confirmation";
 
-$button-stroke: inset 0 0 0 1px;
```

Comment:
Yeah, this one was intentionally removed, because that variable is never referenced.